src/components/opinion/opinionArticle.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import HeaderImage from '../shared/headerImage';
+import ArticleSeries from '../news/articleSeries';
+import OpinionHeadline from '../opinion/opinionHeadline';
+import ArticleStandfirst from '../news/articleStandfirst';
+import OpinionByline from './opinionByline';
+import ArticleBody from '../shared/articleBody';
```

Comment:
Worth using absolute paths for these? `components/shared/...`, `components/news/...` etc.